packages/seatmaps-client/package.json

```diff
@@ -97,10 +106,12 @@
     "jsdom": "^22.1.0",
     "msw": "^1.3.2",
     "postcss": "8.4.31",
+    "process": "^0.11.10",
```

Comment:
Not sure why this is needed. Installed automatically by your ide?

Reply:
this is added due ti vite config build as the build was not working